test/suites/drift/suite_test.go

```diff
@@ -575,53 +575,6 @@ var _ = Describe("Drift", func() {
 		env.ConsistentlyExpectNodeClaimsNotDrifted(time.Minute, nodeClaim)
 	})
 	Context("Failure", func() {
```

Comment:
Makes sense this test doesn't work anymore since the backing of an invalid vnet gets caught.

Maybe leave the test case empty with a TODO instead, since it does seem like a test worth having?